CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Pass index settings to system ingest processor factories. ([#18708](https://github.com/opensearch-project/OpenSearch/pull/18708))
 - Include named queries from rescore contexts in matched_queries array ([#18697](https://github.com/opensearch-project/OpenSearch/pull/18697))
 - Add the configurable limit on rule cardinality ([#18663](https://github.com/opensearch-project/OpenSearch/pull/18663))
+- [Experimental] Add a feature flag to start OpenSearch without joining a cluster ([#18479](https://github.com/opensearch-project/OpenSearch/pull/18479))
```

Comment:
I like this version without the feature flag, but now the changelog, PR title, and commit message need to be updated as well